chatcommands.py

```diff
@@ -376,7 +376,7 @@ def brownie():
     return "Brown!"
 
 
-COFFEES = ['Espresso', 'Macchiato', 'Ristretto', 'Americano', 'Latte', 'Cappuccino', 'Mocha', 'Affogato']
+COFFEES = ['Espresso', 'Macchiato', 'Ristretto', 'Americano', 'Latte', 'Cappuccino', 'Mocha', 'Affogato', 'jQuery']
```

Comment:
Why do you keep adding extra teas and coffees?

Reply:
> Just a fan of jokes and the jQuery meme.

I'm sorry, but the jokes aren't needed. Especially when such joke involves inserting a programming framework into a list of teas. We've decided in the past that we aren't going to add/modify existing joke commands.